config/application_config.go

```diff
@@ -37,8 +37,9 @@ type ApplicationConfig struct {
 	Owner        string `default:"dubbo-go" yaml:"owner" json:"owner,omitempty" property:"owner"`
 	Environment  string `yaml:"environment" json:"environment,omitempty" property:"environment"`
 	// the metadata type. remote or local
-	MetadataType string `default:"local" yaml:"metadata-type" json:"metadataType,omitempty" property:"metadataType"`
-	Tag          string `yaml:"tag" json:"tag,omitempty" property:"tag"`
+	MetadataType        string `default:"local" yaml:"metadata-type" json:"metadataType,omitempty" property:"metadataType"`
+	Tag                 string `yaml:"tag" json:"tag,omitempty" property:"tag"`
+	MetadataServicePort int    `default:"20001" yaml:"metadata_service_port" json:"metadata_service_port,omitempty" property:"metadata_service_port"`
```

Comment:
should not set a default value, when use it you don't known is user set it or default value used